### PR TITLE
docs: docs/jobs/*

### PR DESCRIPTION
**Pull request by xiashuyuan**
Hi,

    I read your job description carefully and update some words in unremarkable place. 

    Wish it could be closer to the habits of English speakers.

Best Wishes.

### CODE REVIEW

**Comment by CJSS** [comment]
I have made some comments for review on this PR.

**Comment by CJSS** [comment]
There are still some changes required as per my previous comments.

**Review by CJSS — CHANGES_REQUESTED** [review]
There are still some changes required as per my previous comments.

File: docs/jobs/affine.pro-remote.md [hunk 1]
```diff
@@ -1,25 +1,26 @@
...
+-   In AFFiNE, developers are self-organized individual engineers who are also responsible team members no matter they are on-site or working remotely.
 
 ### What we are looking for:
 
 -   Software engineering experience with **editor** or **graphics** with professional real-world use cases.
--   Experience and proficiency in **TypeScript** and a **second programming language** preferably **Rust**.
--   You have strong communication and writing skills in English.
--   You are comfortable working in a diverse and cross-functional team.
--   You love the spirit of open source, share our visions and work under those values.
+-   Experience and proficiency in **JavaScript** and a **second programming language** preferably **Rust**.
```
[the remark this answers is not in the document]
Comment (xiashuyuan) [reply docs/jobs/affine.pro-remote.md:20]: Sorry, it is a mistake.

File: docs/jobs/affine.pro-remote.md [hunk 2]
```diff
@@ -1,25 +1,26 @@
...
 
 ### This position is for:
 
--   Developing AFFiNE **the open source way**, including coding and community engagement.
--   Researching and supporting onboarding of new use cases AFFiNE.pro subscribers.
--   Improving our **block editor** and **graphics editor**.
--   Assisting our subscribers in utilizing our product in a data-based way with help from the operational teams.
--   Researching on better activation of potential subscribers. At AFFiNE, developers are self-organized individual engineers who are also responsible team members whether they are on-site or working remotely.
+-   Develop AFFiNE in **the open source way**, including code and community engagement.
+-   Research and support **onboarding process** of new use cases for AFFiNE.pro subscribers.
```
[the remark this answers is not in the document]
Comment (xiashuyuan) [reply docs/jobs/affine.pro-remote.md:11]: You are right

File: docs/jobs/affine.pro-remote.md [hunk 3]
```diff
@@ -1,25 +1,26 @@
...
 
 -   Software engineering experience with **editor** or **graphics** with professional real-world use cases.
--   Experience and proficiency in **TypeScript** and a **second programming language** preferably **Rust**.
--   You have strong communication and writing skills in English.
--   You are comfortable working in a diverse and cross-functional team.
--   You love the spirit of open source, share our visions and work under those values.
+-   Experience and proficiency in **JavaScript** and a **second programming language** preferably **Rust**.
+-   Strong communication and writing skills in English.
+-   Work in a diverse and cross-functional team with skill and ease.
+-   Love open source spirit, share our visions and work under those values.
```
[the remark this answers is not in the document]
Comment (xiashuyuan) [reply docs/jobs/affine.pro-remote.md:23]: Correct, the title ends with 'for'.

File: docs/jobs/mysc.app.md [hunk 4]
```diff
@@ -5,17 +5,17 @@
 -   Build APIs in the Data Platform to support new capabilities within mysc.
 -   Work with backend and client side databases (MongoDB, Redis, SQLite)
 -   Design and implement algorithms that are highly performant, resilient against failures and race conditions and are easy to use by application developers
--   Build up solid knowledge of our product to understand end to end system behaviour and data flow
--   Execute performance profiling on existing systems to understand key bottlenecks and improve on their performance characteristics
+-   Build up solid knowledge of our product to understand end to end system behavior and data flow
+-   Execute performance profiling on existing systems to figure out key bottlenecks and improve their performance characteristics
```
[the remark this answers is not in the document]
Comment (xiashuyuan) [reply docs/jobs/mysc.app.md:9]: I agree 'identify' is better

**Comment by CJSS** [comment]
Thanks for your contribution - your PR will be included with edits in #494.

**Pull request closed without merging**